actionview/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+*   Add the template digest to the cache key for collection caching.
```

Comment:
This isn't entirely true. It works in real apps, I've seen the cache keys contain the digest there.

The tests render the collection from the controller and I think that's what screwing this up.